src/components/Calendar/index.js

```diff
@@ -12,7 +12,81 @@ export default function Calendar({
   capture,
   isAutosized
 }) {
-  const [{ pinnedCrns }] = useContext(TermContext);
+  const [{ pinnedCrns, oscar }] = useContext(TermContext);
+
+  const dayMap = DAYS.reduce((acc, day) => {
```

Comment:
It would be useful to include a comment about what exactly this data structure contains